app/services/drug_names_service.rb

```diff
@@ -0,0 +1,35 @@
+class DrugNamesService
+  def find_generic_drugs(concept_set_id)
```

Comment:
i did not see that function, i think i will use the already exiting function (find_all_by_concept_set)

Reply:
This method (find_all_by_concept_set) is used to get drug list, so i have used it when getting OPD drug list but the method (find_generic_drugs) it is used to get general drug names so i have left it as it is since i am not seeing any already made function for getting general drug names